DirectProgramming/C++SYCL_FPGA/Tutorials/Features/multi_aocx/device_code_split/README.md

```diff
@@ -0,0 +1,247 @@
+# Using the flows test 'device_code_split'
+
+This FPGA tutorial demonstrates the -fsycl-device-code-split=per_source|per_kernel
+
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | Linux* Ubuntu* 18.04/20.04 <br> RHEL*/CentOS* 8 <br> SUSE* 15 <br> Windows* 10
+| Hardware                          | Intel&reg; Programmable Acceleration Card (PAC) with Intel Arria&reg; 10 GX FPGA <br> Intel&reg; FPGA Programmable Acceleration Card (PAC) D5005 (with Intel Stratix&reg; 10 SX) <br> Intel&reg; FPGA 3rd party / custom platforms with oneAPI support <br> **Note**: Intel&reg; FPGA PAC hardware is only compatible with Ubuntu 18.04*
+| Software                          | Intel® oneAPI DPC++/C++ Compiler
+| What you will learn               | How to use the `-fsycl-device-code-split=per_source|per_kernel` to create multiple aocx.
+| Time to complete                  | 30 minutes
+
+> **Note**: Even though the Intel DPC++/C++ OneAPI compiler is enough to compile for emulation, generating reports and generating RTL, there are extra software requirements for the simulation flow and FPGA compiles.
+>
+> For using the simulator flow, one of the following simulators must be installed and accessible through your PATH:
+> - Questa*-Intel® FPGA Edition
+> - Questa*-Intel® FPGA Starter Edition
+> - ModelSim® SE
+>
+> When using the hardware compile flow, Intel® Quartus® Prime Pro Edition must be installed and accessible through your PATH.
+
+## Purpose
+
+`-fsycl-device-code-split=[per_kernel|per_source]` flag could be used when a design doesn't fit a single FPGA, so user must split it into multiple smaller images.
+
+### Simple Code Example
+
+```
+icpx ... simple.cpp -o simple.exe -fsycl-device-code-split=per_kernel
+icpx ... -reuse-exe=simple.exe -Xsv
+```
+
+### Using the device_code_split
+
+- Split a large design into smaller images
+  - `per_kernel` - a separate device code module is created for each SYCL kernel
+  - `per_source` - a separate device code module is created for each source
+- Users don't need to restructure their code 
```

Comment:
I think you can elaborate more on this:
_Users do not need to manually restructure their code or source files to get smaller hardware image files. This flag allows the compiler to do so automatically._